### DISCUSSION TITLE
Archive EarTrumpet

### DISCUSSION THREAD

**Discussion by ChaseKnowlden** [File-New-Project/EarTrumpet#1512] [Q&A] [answered]
Volume mixer is built into Windows, which has the same functionality as EarTrumpet.

**Comment by riverar** [comment, marked as answer]
Nope.

**Comment by Chaython** [comment]
> Volume mixer is built into Windows, which has the same functionality as EarTrumpet.

In Windows 11 [Win][CTRL][V] brings up the volume mixer, but you cannot resize it, the tiny window that appears is rather unusable and I like being able to see the mixer by clicking the EarTrumpet icon rather than hotkey so I can one hand....
Maybe for windows 11+ an app that manipulates the built in volume mixer would be better since, you can't remove the tray volume icon anymore etc....
Also running the windows 7 era volume mixer Win + R -> "sndvol.exe" -> OK -> Pin to taskbar, selecting the device drop down doesn't switch the device, the GUI is ugly and solid white....

**Comment by Gravifer** [comment]
A bit ignorant, no?
